### ISSUE TITLE
'yotta version' doesn't work in Windows

### ISSUE THREAD

**Issue by bogdanm** [ARMmbed/yotta#252]
```
> yt version patch
error: command failed: ['git', '--work-tree=c:\\work\\mbed\\ns_with_minar\\minar-platform-k64f', '--git-dir=c:\\work\\mbed\\ns_with_minar\\minar-platform-k64f\\.git', 'add', 'c:\\work\\mbed\\ns_with_minar\\minar-platform-k64f\\module.json']:fatal: unable to access '../../../../../c:\work\mbed\ns_with_minar\minar-platform-k64f\.git/config': Invalid argument
```

**Comment by autopulated** [comment]
This is in a plain `cmd.exe` shell, right? (not in git-bash/msys, posh-git or cygwin?)

**Comment by bogdanm** [comment]
Just tried in `cmd.exe`, same thing.

**Comment by autopulated** [comment]
ack

**Comment by autopulated** [comment]
this was fixed in #262
